src/components/activity-page/managed-interactive/iframe-runtime.tsx

```diff
@@ -85,12 +85,13 @@ export const IframeRuntime: React.ForwardRefExoticComponent<IProps> = forwardRef
 
   const [ heightFromInteractive, setHeightFromInteractive ] = useState(0);
   const [ ARFromSupportedFeatures, setARFromSupportedFeatures ] = useState(0);
-  const [resetInteractive, setResetInteractive] = useState(false);
   const [reloadCount, setReloadCount] = useState<number>(0);
+  const [iframePhoneTimeout, setIframePhoneTimeout] = useState<NodeJS.Timeout|undefined>(undefined);
```

Comment:
```suggestion
  const [iframePhoneTimeout, setIframePhoneTimeout] = useState<number|undefined>(undefined);
```
since we're running in the browser

Reply:
Thanks, @kswenson. Using `number` resulted in an error:

`Argument of type 'Timeout' is not assignable to parameter of type 'SetStateAction<number | undefined>'.`

Does the latest change to line 89 seem like the right solution?

Reply:
It looks better because it's not explicitly wrong, but it's still implicitly wrong. If you hover over the value you can still see that it has type `NodeJS.Timeout | undefined`, but at runtime we know that it's the browser version of `setTimeout` that will be called and that the return value will be a number, so using `Node.JS.Timeout` is just telling the TypeScript compiler to type-check against the wrong type. The optimal solution would be to figure out what's introducing the node.js types into the application (I tried removing `node` from the `types` in the `tsconfig.json` and it didn't solve the problem, which suggests the types are being inserted somewhere _in addition_ to the `tsconfig.json`). I think the next-best solution would be to use `number` and then use `window.setTimeout` at the point of call, which is inelegant but at least correct.

Reply:
OK, I see. I've made the changes you suggested.